### PR TITLE
Performance Data Re-Entry Patch

### PR DESCRIPTION
**Pull request by mackgalante**
- Updating all vignettes' performance data from templates
- Entering performance values that should pass comparator needs (if comparators are entered in persona spec spreadsheet)
- Updating tracking sheet as commits take place

### CODE REVIEW

File: vignettes/social_gain_vignette/readme.md [hunk 1]
```diff
@@ -14,25 +14,25 @@ MPOG calculates performance benchmarks and averages for each institution. One co
...
-|Feb  |              94%|               97|          87|
-|Mar  |              97%|               95|          86|
+|Month|Performance Level|Peer Average|75th Percentile Benchmark|90th Percentile Benchmark|
+|-----|-----------------|------------|-------------------------|-------------------------|
+|Jul  |              90%|          90|                       92|                       96|
+|Aug  |              90%|          90|                       92|                       96|
+|Sept |              90%|          90|                       92|                       96|
+|Oct  |              90%|          90|                       92|                       96|
+|Nov  |          **93%**|          90|                       92|                       96|
+|Dec  |          **97%**|          90|                       92|                       96|
```
Comment (mackgalante) [inline vignettes/social_gain_vignette/readme.md:35]: Comparators:
- [x] 90th percentile
- [ ] 75th percentile
- question, we want preferences to determine that 75th percentile social gain isn't going to be an accepted message candidate, correct?

File: vignettes/social_better_vignette/readme.md [hunk 2]
```diff
@@ -14,25 +14,25 @@ MPOG calculates performance benchmarks and averages for each institution. One co
...
-|Dec  |              90%|               98|          94|
-|Jan  |              92%|               96|          90|
+|Month|Performance Level|Peer Average|75th Percentile Benchmark|90th Percentile Benchmark|
+|-----|-----------------|------------|-------------------------|-------------------------|
+|Jul  |              90%|          90|                       92|                       96|
+|Aug  |              90%|          90|                       92|                       96|
+|Sept |              90%|          90|                       92|                       96|
+|Oct  |              90%|          90|                       92|                       96|
+|Nov  |          **88%**|          90|                       92|                       96|
+|Dec  |          **98%**|          90|                       92|                       96|
```
Comment (mackgalante) [inline vignettes/social_better_vignette/readme.md:24]: Reformatted based on new template document I cooked up. Are values appropriate?

Comparators:
- [x] 90th percentile (candidate acceptable)
- [x] peer average

File: vignettes/social_better_vignette/readme.md [hunk 3]
```diff
@@ -14,25 +14,25 @@ MPOG calculates performance benchmarks and averages for each institution. One co
...
-|Dec  |              91%|               98|          92|
-|Jan  |              93%|               96|          91|
+|Month|Performance Level|Peer Average|75th Percentile Benchmark|90th Percentile Benchmark|
+|-----|-----------------|------------|-------------------------|-------------------------|
+|Jul  |              85%|          85|                       88|                       92|
+|Aug  |              85%|          85|                       88|                       92|
+|Sept |              85%|          85|                       88|                       92|
+|Oct  |              85%|          85|                       88|                       92|
+|Nov  |          **83%**|          85|                       88|                       92|
+|Dec  |          **87%**|          85|                       88|                       92|
```
Comment (mackgalante) [inline vignettes/social_better_vignette/readme.md:35]: Comparators:
- [ ] 90th percentile
- [x] peer average

**Comment by mackgalante** [comment]
Above commits are more of the same as the above, however these are still placeholders as we do not have comparators (what thresholds the data should cross) assigned in the persona spec sheet for vignettes other than social gain and better. 

Updated all existing vignettes to at least have template values, with appropriate changes to be acceptable for the CP that they are examples of. Can change later.

**Comment by 00hchung99** [comment]
Duplicate tables so there is one for each persona within a role (eg. separate attending table for Alice and Fahad, which have the same performance data besides the last couple months, but different benchmarks and averages since they are from different institutions)

**Pull request merged**